other-examples/collector/docker-nr-infra/README.md

```diff
@@ -0,0 +1,98 @@
+# Monitoring Application  with OpenTelemetry Collector and Docker with NR infra agent
+
+This is a simple application instrumented with OpenTelemetry JavaScript. It demonstrates how to configure OpenTelemetry JavaScript to send data to New Relic.
+
+Additionally, it demonstrates correlating OpenTelemetry entities with docker.
+
+
+## Requirements
+
+* A linux machine with docker daemon and docker compose (docker stats receiver only supports Linux).
+* [A New Relic account](https://one.newrelic.com/)
+* [A New Relic license key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#license-key)
+
+## Running the example
+
+  1. Update the `NEW_RELIC_API_KEY` value with to your New Relic license key.
+
+      * [newrelic-infra.yaml](newrelic-infra/newrelic-infra.yaml)
+
+      ```yaml
+        # New Relic API key to authenticate the export requests.
+        # docs: https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#license-key
+      license_key: <NEW_RELIC_API_KEY>
+      ```
+
+      * [otel-config.yaml](./otel-config.yaml)
+
+      ```yaml
+      # ...omitted for brevity
+        # New Relic API key to authenticate the export requests.
+        # docs: https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#license-key
+        otlp:
```

Comment:
New Relic recommends the [http/protobuf](https://docs.newrelic.com/docs/opentelemetry/best-practices/opentelemetry-otlp/#configure-endpoint-port-protocol) version of OTLP.

```suggestion
        otlphttp:
```